### DISCUSSION TITLE
Error al clonar TP1

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#45] [Preguntas y respuestas] [answered]
Hola, me apareció este error cuando intentaba clonar el TP1. Espero sugerencias, desde ya gracias. 

![Prog-1-](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/bd68ccff-bfbf-434f-a500-48e1114ef3f6)

**Comment by martinvilu** [comment, marked as answer]
No es un error, es una advertencia, lo que está al final indica que el `clone` termino exitosamente.
